.travis.yml

```diff
@@ -39,11 +40,47 @@ env:
   - DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
   - DB=pgsql MOODLE_BRANCH=MOODLE_36_STABLE
   - DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
+  - DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
```

Comment:
If you add more combinations to travis it can slow it right down. Your test run was:

 Ran for 29 min 51 sec
 Total time 1 hr 55 min 20 sec

There was 52 builds in the matrix. We want to make sure we have covered every db, php version and stable, but we don't need to test every combination of all of the above. We want to get the build cycle back to around 20 mins and keep it there or less.

So can you please weed out some of the combinations which are less interesting, perhaps keep all the LTS versions ie 2.7, 3.1 and 3.5 and then everything after 3.5

Reply:
Feel free to do that in a follow up issue too

Reply:
updated